packages/app/src/messages/services/thread.service.ts

```diff
@@ -163,6 +163,54 @@ export class ThreadService {
     }
   }
 
+  public async toggleThreadReadStatus(
+    threads: Thread[]
+  ): Promise<RequestResponse<Thread[]>> {
```

Comment:
And it

Reply:
Fixed in [74a6af8](https://github.com/mudita/mudita-center/pull/728/commits/74a6af816f864d38862a81f9539521a7f38d27b0)